sirbot_pyslackers/endpoints/slack/messages.py

```diff
@@ -13,8 +13,15 @@
 from .utils import ADMIN_CHANNEL, HELP_FIELD_DESCRIPTIONS
 
 LOG = logging.getLogger(__name__)
-STOCK_REGEX = re.compile(r"\b(?P<asset_class>[cs])\$(?P<symbol>\^?[A-Z.]{1,5})\b")
+STOCK_REGEX = re.compile(
+    r"\b(?P<asset_class>[cs])\$(?P<symbol>\^?[A-Z.]{1,5})(-(?P<currency>[A-Z]{3}))?\b"
```

Comment:
Instead of making it another capture group, can you make it just a grouping: `(?:-(?P<currency>...))?` That's just a tiny bit more clear in the usage later.